### PR TITLE
Update configuration for Cosmos

### PR DESCRIPTION
**Pull request by jsancio**
## What type of changes does your doc introduce?
- [ ] Bug fix
- [x] New feature 

## What is the urgency level of this change?
- [ ] Blocker
- [x] High
- [ ] Medium

## I have completed these items:
- [x] I have tested all commands and code snippets.
- [ ] I have built locally and tested for broken links and formatting.
- [x] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9). It only affects 1.9.
- [x] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

Ping `@emanic`, `@sascala`, or `@joel-hamill` for reviewing pull request changes.

## If you included a comment with your commit, it appears here:
Relates to this PR: https://github.com/dcos/dcos/pull/974

### CODE REVIEW

File: 1.9/administration/installing/custom/configuration-parameters.md [hunk 1]
```diff
@@ -234,6 +234,17 @@ This parameter specifies whether to enable sharing of anonymous data for your cl
 
 If you’ve already installed your cluster and would like to disable this in-place, you can go through an [upgrade][3] with the same parameter set.
 
+### cosmos_config
+This parameter specifies a dictionary of packaging configuration to pass to Cosmos, DC/OS package
+manager. If set, both of the following options must be set.
```
Comment (joel-hamill) [inline 1.9/administration/installing/custom/configuration-parameters.md]: ...`the following options must be specified.`

File: 1.9/administration/installing/custom/configuration-parameters.md [hunk 2]
```diff
@@ -234,6 +234,17 @@ This parameter specifies whether to enable sharing of anonymous data for your cl
 
 If you’ve already installed your cluster and would like to disable this in-place, you can go through an [upgrade][3] with the same parameter set.
 
+### cosmos_config
+This parameter specifies a dictionary of packaging configuration to pass to Cosmos, DC/OS package
```
Comment (joel-hamill) [inline 1.9/administration/installing/custom/configuration-parameters.md]: ...`packaging configuration to pass to the [DC/OS package manager](https://github.com/dcos/cosmos).

File: 1.9/administration/installing/custom/configuration-parameters.md [hunk 3]
```diff
@@ -234,6 +234,17 @@ This parameter specifies whether to enable sharing of anonymous data for your cl
...
+manager. If set, both of the following options must be set.
+
+* **staged_package_storage_uri**
+  This parameter specifies the location where you would like to temporarily store DC/OS Package
```
Comment (joel-hamill) [inline 1.9/administration/installing/custom/configuration-parameters.md]: ...`specifies where to temporarily store DC/OS packages while they are being added.`

File: 1.9/administration/installing/custom/configuration-parameters.md [hunk 4]
```diff
@@ -234,6 +234,17 @@ This parameter specifies whether to enable sharing of anonymous data for your cl
...
+  while the are being added. The value must be a file URL. E.g.
+  `file:///var/lib/dcos/cosmos/staged-packages`
+* **package_storage_uri**
+  This parameter specifies the location where you would like to permanetly store DC/OS Package.
```
Comment (joel-hamill) [inline 1.9/administration/installing/custom/configuration-parameters.md]: `specifies where to permanently store DC/OS packages.`

File: 1.9/administration/installing/custom/configuration-parameters.md [hunk 5]
```diff
@@ -234,6 +234,17 @@ This parameter specifies whether to enable sharing of anonymous data for your cl
...
+  The value must be a file URL. E.g. `file:///var/lib/dcos/cosmos/packages`
```
Comment (joel-hamill) [inline 1.9/administration/installing/custom/configuration-parameters.md]: `The value must be a file URL, for example...`

**Comment by jsancio** [comment]
Back to you @joel-hamill

**Comment by joel-hamill** [comment]
@jsancio can provide what a full example specification looks like, e.g. 
```
cosmos_config:
staged_package_storage_uri: file:///var/lib/dcos/cosmos/staged-packages
package_storage_uri: file:///var/lib/dcos/cosmos/packages
```

might be good to add to one of the examples at the bottom of this file.

**Comment by jsancio** [comment]
@joel-hamill Done. Back to you. Thanks!

**Comment by joel-hamill** [comment]
@jsancio this content looks good. the final step with this topic is to make this change in the enterprise repo as well (https://github.com/mesosphere/dcos-docs-enterprise/blob/master/1.9/administration/installing/custom/configuration-parameters.md). This particular topic is not shared between repos because it is different between Ent and OSS.

**Pull request merged**